packages/resizin/src/upload/upload.ts

```diff
@@ -11,35 +34,40 @@ const uploadImage = (serverUrl: string, apiKey: string, imageId: string, file: s
             throw new Error('API KEY is missing!');
         }
         if (!imageId) {
-            throw new Error('Name is missing!'); // TODO - change to "Id is missing"
+            throw new Error('Image id is missing!');
         }
         if (!file) {
-            throw new Error('Body is missing!'); // TODO - change to "File is missing"
+            throw new Error('File is missing!');
         }
 
-        const options: any = {
+        const options = defaults(uploadOptions, { fileType: type.IMAGE });
+        const fileType = options.fileType === type.IMAGE ? options.fileType : type.FILE;
+
+        const fetchOptions: any = {
             method: 'POST',
             headers: {
                 Authorization: 'Key ' + apiKey,
             },
         };
 
-        // TODO - polyfill FormData in node-resizing package
-        let formData;
-        if (typeof window === 'undefined') {
-            formData = new NodeFormData();
-        } else {
-            formData = new FormData();
-        }
+        const formData = new FormData();
```

Comment:
this FormData are the global one transformed above in node part?

Reply:
Depends on if it's used as a standalone package `@ackee/resizin` or through a node package `@ackee/node-resizin`.